opencog/data/table/table.cc

```diff
@@ -44,6 +44,7 @@
 #include "opencog/combo/combo/simple_nn.h"
 #include "opencog/combo/combo/convert_ann_combo.h"
 #include "opencog/utils/value_key.h"
+#include "opencog/utils/valueUtils.h"
```

Comment:
I didn't realize all these were with quotes. It's not a big deal obviously, but according to https://wiki.opencog.org/w/Development_standards#Includes as well as https://ardour.org/images/therules.jpg it should be fixed! ;-)